src/pages/booklist/containers/BooklistContainer.tsx

```diff
@@ -1,26 +1,23 @@
 import { useEffect } from 'react';
 
-import { Box, Container, Divider, Heading } from '@chakra-ui/layout';
+import { Box } from '@chakra-ui/layout';
 import { Center, Spinner, Text, VStack } from '@chakra-ui/react';
 import { useParams } from 'react-router';
 
 import { Term } from 'lib/fetchers';
-import { useDarkMode } from 'lib/hooks/useDarkMode';
 import { logEvent } from 'lib/utils/logEvent';
 import { getReadableTerm } from 'lib/utils/terms';
 
 import { Page } from 'common/layouts/Page';
+import { NotFound } from 'common/notfound/NotFound';
```

Comment:
```suggestion
import { NotFound } from 'common/notFound/NotFound';
```
what I mean by this is renaming the folder, not just the change in the code